units/en/unit2/smolagents/multi_agent_systems.mdx

```diff
@@ -0,0 +1,317 @@
+# Multi-Agent Systems
+
+Multi-agent systems enable specialized agents to collaborate on complex tasks, improving modularity, scalability, and robustness. Instead of relying on a single agent, tasks are distributed among agents with distinct capabilities.
```

Comment:
```suggestion
Multi-agent systems enable **specialized agents to collaborate on complex tasks**, improving modularity, scalability, and robustness. Instead of relying on a single agent, tasks are distributed among agents with distinct capabilities.
```